### PR TITLE
wakeonlan: add page

### PR DESCRIPTION
**Pull request by Renner0E**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
wakeonlan 0.42

### CODE REVIEW

File: pages/common/wakeonlan.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# wakeonlan
+
+> wakeonlan - sends 'magic packets' to wake-on-lan (WOL) enabled PCs.
```
Comment (spageektti) [inline pages/common/wakeonlan.md:3]:
```suggestion
> Send packets to wake-on-lan (WOL) enabled PCs.
```
See style guide.

File: pages/common/wakeonlan.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+> See also: `wakeonlan`.
+> More information: <https://github.com/jpoliv/wakeonlan>
```
Comment (spageektti) [inline pages/common/wakeonlan.md:5]:
```suggestion
> More information: <https://github.com/jpoliv/wakeonlan>.
```

File: pages/common/wakeonlan.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+- Send magic packet to a specific device via IP address:
+
+`wakeonlan {{01:02:03:04:05:06}} -i {{192.168.178.2}}`
+
+- Print the commands, but don't execute them (dry-run):
+
+`wakeonlan -n {{01:02:03:04:05:06}}`
+
+- quiet mode:
```
Comment (gutjuri) [inline pages/common/wakeonlan.md:19]:
```suggestion
- Run in quiet mode:
```

**Review by gutjuri — COMMENTED** [review]
Thanks for your contribution! I have one suggestion which would fix the linter error :)

File: pages/common/wakeonlan.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+> Send packets to wake-on-lan (WOL) enabled PCs.
+> See also: `wakeonlan`.
+> More information: <https://github.com/jpoliv/wakeonlan>.
+
+- Send magic packets to all devices on the local network (255.255.255.255) by specifying a MAC address:
+
+`wakeonlan {{01:02:03:04:05:06}}`
+
+- Send magic packet to a specific device via IP address:
```
Comment (spageektti) [inline pages/common/wakeonlan.md:11]:
```suggestion
- Send packet to a specific device via IP address:
```

File: pages/common/wakeonlan.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# wakeonlan
+
+> Send packets to wake-on-lan (WOL) enabled PCs.
+> See also: `wakeonlan`.
+> More information: <https://github.com/jpoliv/wakeonlan>.
+
+- Send magic packets to all devices on the local network (255.255.255.255) by specifying a MAC address:
```
Comment (spageektti) [inline pages/common/wakeonlan.md:7]:
```suggestion
- Send packets to all devices on the local network (255.255.255.255) by specifying a MAC address:
```

**Review by spageektti — APPROVED** [review]
LGTM. Thanks!

File: pages/common/wakeonlan.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# wakeonlan
+
+> Send packets to wake-on-lan (WOL) enabled PCs.
```
Comment (spageektti) [inline pages/common/wakeonlan.md:3]:
```suggestion
> Send packets to wake-on-LAN (WOL) enabled PCs.
```

File: pages/common/wakeonlan.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+> See also: `wakeonlan`.
```
Comment (spageektti) [inline pages/common/wakeonlan.md:4]:
```suggestion
```
See also is not correct as `wakeonlan` is the current page.

**Review by spageektti — APPROVED** [review]

**Review by gutjuri — APPROVED** [review]

**Pull request merged**